back/spec/factories/projects.rb

```diff
@@ -418,7 +427,7 @@
         end
       end
 
-      factory :private_groups_single_phase_project do
+      factory :private_groups_project do
```

Comment:
Changed merged :private_groups_single_phase_project amd :private_groups_project so that all private groups projects now have a phase